config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presets.yaml

```
@@ -38,3 +38,30 @@ presets:
   - mountPath: /var/run/secrets/azure-token/serviceaccount
     name: azure-token
     readOnly: true
+        key: serviceAccountSigningKey
+- labels:
+    preset-azure-community-wi: "true"
+  env: # below env values are not secrets
+  - name: AZURE_CLIENT_ID # AZURE_CLIENT_ID is being overloaded with Azure Workload ID
```

```suggestion
  - name: AZURE_CLIENT_ID 
```

I think that comment was from a previous iteration